cron/daily/populate_search_index.js

```diff
@@ -45,10 +45,18 @@ const done = (error) => {
 const populateIndex = async () => {
   const collectives = await models.Collective.findAll({
     where: {
-      type: collectiveTypes.COLLECTIVE,
+      type: {
+        [Op.or]: [collectiveTypes.COLLECTIVE, collectiveTypes.ORGANIZATION],
+      },
       id: {
         [Op.notIn]: [1, 7]
-      }
+      },
+      slug: {
+        [Op.or]: {
+          [Op.notLike]: '%\_old',
+          [Op.notLike]: '%inactive',
```

Comment:
@xdamman They are not soft deleted. I'm fine with cleaning up these records by soft deleting them, example -> https://opencollective.com/githubstandards_old. 

We can't rely on just "inactive" for this because we have collectives that have not been approved by their host that should still show in search.

Reply:
Right.
The problem with soft delete is that it may break things in other places if we forget to also soft delete all the rows in the `Members` table for example. So we need to be cautious with that. That's why we didn't delete them and just renamed them (but that's not very sustainable). Give it a try and let's make sure it doesn't break anything :-)

Reply:
@xdamman Ok I'll update this PR to remove the additional slug query. I'm looking into how to automatically delete dependent records through Sequelize.